### ISSUE TITLE
Dark Theme w/ light text in OS makes toolbar and menus almost invisible !

### ISSUE THREAD

**Issue by m4udSec** [giuspen/cherrytree#859]
how can i change the toolbar color to dark blue to go with the rest of theme ? as of now i change evertyhing but toolbar and menus remains light colored ! and the way it is, the problem is the light text agains the light backgroung !!  its barely to not visible at all, to read the menu in cherry tree, do i NEED to have a light themed OS ! light themes are horrible for working !
thank you in advance for the help !
i have the latest cherrytree version !

**Comment by txe** [comment]
Hi, your cherrytree is python or c++? and can you give a screenshot?

**Comment by m4udSec** [comment]
![image](https://user-images.githubusercontent.com/56730389/83190257-874cf780-a108-11ea-893f-2581c667c673.png)

**Comment by txe** [comment]
Ok, good. I was afraid it was a snap, it wouldn't be so easy to change.
I'll give a solution in a day or two. I can forget, so don't be afraid to ping me.

**Comment by m4udSec** [comment]
thank you bud, you guys are awesome :) i freaking love cherrytree, i`ll be in touch ! 
cheers m8

**Comment by txe** [comment]
Hi, I thought you couldn't change toolbar color, but I re-read your issue again,
so you can change the toolbar color to blue by configuring the theme, but the actual issue is with the text color, am I right?

**Comment by m4udSec** [comment]
bud, I couldn`t make this toolbar change, it was stuck in this grayish color, and I also had this menus issue, but, it didn't matter whether i change in both place in preference the themes to be dark,toolbar color would just be gray, specifically only the toolbar and the menus , but after a few uninstalls and reinstalls, the toolbar color finally has changed to dark blue as you said above, but i can swear that, was unexpected, because the "theme" fucntionallity" does not seem to involve the toolbar at all, my collegues that also works with Cherrytree, can not replicate my new dark blue toolbar at their Cherrytree !
In conclusion, the picture above has Cherrytree set at both options to "Dark" but.. it doesn't turn the toolbar into dark blue, it leaves it in gray, so if you also have an OS dark theme, then your icons won't be showing in Cherrytree toolbar, and the menus will be unintelligible since its white on white! 
so, my guess is, if that functionality exists, it straights doesn't work! i had 6 machines testing it, to see if i could change the damned toolbar color, but ..nope, sadly couldn't !
anyways, thank you very much for your attention, bud ! if any feedback towards it, i would be happy to hear ! cheers

**Comment by hellerbarde** [comment]
@m4udSec  use gtk-chtheme to switch to a proper dark GTK2 theme. your problems will be solved
